.github/workflows/main.yml

```
@@ -0,0 +1,38 @@
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  test:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
+    with:
+      posargs: --openmp-expected=True
+      coverage: codecov
+      envs: |
+        # Code style
+        - linux: style
+        # Docs
+        - linux: build_docs
+        # Standard tests
+        - linux: py36-test-oldestdeps
+        - linux: py37-test
+        - linux: py38-test
+        - linux: py38-test-dev
+        - macos: py38-test-dev
+          posargs: --openmp-expected=False
+        - windows: py36-test
+        - windows: py38-test-dev
+        # Test with more compilers, for the OpenMP helpers
+        - macos: py38-test-osxclang-conda
+        - linux: py38-test-linuxgcc-conda
+        # Test downstream packages
+        - linux: py39-downstream
+
```

This should make it use the windows-2019 image on Windows:
```suggestion
    env:
      OA_WINDOWS_RUNNER: windows-2019

```

---

Oops just saw this!